ownCloud.xcodeproj/project.pbxproj

```diff
@@ -4799,8 +4799,8 @@
 				APP_BUILD_FLAGS = "$(inherited)";
 				APP_BUILD_FLAGS_SWIFT = "$(APP_BUILD_FLAGS)";
 				APP_PRODUCT_NAME = ownCloud;
-				APP_SHORT_VERSION = 11.8.2;
-				APP_VERSION = 208;
+				APP_SHORT_VERSION = 11.8.3;
+				APP_VERSION = 210;
```

Comment:
Versioning should be removed, because it will be also merged into the `milestone/11.9.0`